resources/lta_cmd.py

```
@@ -0,0 +1,205 @@
+    parser_request_new.add_argument("--dest",
+                                    action="append",
+                                    help="destination of bundles",
+                                    required=True)
+    parser_request_new.add_argument("--path",
+                                    help="Data Warehouse path to be transferred",
+                                    required=True)
+    parser_request_new.add_argument("--json",
+                                    help="display output in JSON",
+                                    action="store_true")
+    parser_request_new.set_defaults(func=request_new)
+
+    # define a subparser for the 'request status' subcommand
+    parser_request_status = request_subparser.add_parser('status', help='query transfer request status')
+    parser_request_status.add_argument("--uuid",
+                                       help="identity of transfer request",
+                                       required=True)
+    parser_request_status.add_argument("--json",
+                                       help="display output in JSON",
+                                       action="store_true")
+    parser_request_status.set_defaults(func=request_status)
+
+    # define a subparser for the 'status' subcommand
+    parser_status = subparser.add_parser('status', help='perform a status query')
+    parser_status.add_argument("component",
+                               choices=["picker", "bundler"],
+                               help="optional LTA component",
+                               nargs='?')
+    parser_status.add_argument("--json",
+                               help="display output in JSON",
+                               action="store_true")
+    parser_status.set_defaults(func=status)
+
+    # parse the provided command line arguments and call the function
+    args = parser.parse_args()
+    exit_code = await args.func(args)
+    sys.exit(exit_code)
```

This is considered not-pythonic.  The "correct" way is to just raise an exception and let that return a non-zero exit code.

---

I've removed the exit coding handling, and the try-catch blocks.
Let exceptions raise where they may...